lib/wsdl.js

```diff
@@ -1932,10 +1932,22 @@ WSDL.prototype.findSchemaType = function(name, nsURI) {
   return schema.complexTypes[name];
 };
 
-WSDL.prototype.findChildSchemaObject = function(parameterTypeObj, childName) {
+WSDL.prototype.findChildSchemaObject = function(parameterTypeObj, childName, backtrace) {
   if (!parameterTypeObj || !childName) {
     return null;
   }
+
+  if (!backtrace) {
+    backtrace = [];
+  }
+
+  if (backtrace.indexOf(parameterTypeObj) >= 0) {
+    // We've recursed back to ourselves; break.
+    return null;
+  } else {
+    backtrace = backtrace.concat([parameterTypeObj]);
```

Comment:
I'm always eager to learn new things but I'm afraid I not able to follow you here 😸 

Would it be possible for you to create a jsbin or gist or something like that in order for me to get the difference? Maybe it's because it's already late in the evening (for me) :austria:  I would be very thankful for this 👍 💯